tap_gitlab/streams.py

```diff
@@ -44,15 +128,52 @@ class IssuesStream(ProjectBasedStream):
     bookmark_param_name = "updated_after"
     is_sorted = True
     extra_url_params = {"scope": "all"}
+    schema_filepath = None  # to allow the use of schema below
 
     def post_process(self, row: dict, context: Optional[dict] = None) -> Optional[dict]:
         result = super().post_process(row, context)
         if result is None:
             return None
 
-        result["assignees"] = object_array_to_id_array(result["assignees"])
+        # XXX: breaks backwards compatibility
+        # result["assignees"] = object_array_to_id_array(result["assignees"])
```

Comment:
Can you explain why this is getting removed?

Reply:
I'm changing it from an array of `id`s to an array of `user_object`s in the schema. To be honest, it makes more sense for our use case, but maybe not for everyone, but I feel that it makes sense to keep the info since it's been downloaded anyway. The result still goes into `issues__assignees` it's just the field name that changes (it's now in the `id` column).